### PR TITLE
ROC-3706: Add amended flag when defendant address is updated

### PR DESCRIPTION
**Pull request by rattur**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-3706

### Change description ###

Add text '(as amended)' under the address section when defendant address details updated

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Comment by rattur** [comment]
screen shot 2018-05-10 at 11 50 03

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 1]
```diff
@@ -190,7 +194,10 @@
       </tr>
       {% endif %}
       <tr>
-        <td>Address:</td>
+        <td class="keep-formatting">Address:
+          {% if defendant.addressAmended %}
+          (as amended)
```
Comment (timja-kainos) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: is this where its supposed to go? will this break robotics?
we used to have it after this address first line I think

Comment (rattur) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: That's what Marc James and Art would like

Comment (timja-kainos) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: will it break robotics?

Comment (rattur) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: I'm not sure Tim, will need to check

Comment (geek-soft) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: @rattur I managed to find previous version. Here is what we had before https://github.com/hmcts/cmc-claim-store/pull/279/files#diff-5b3cba7f1066682c02d7992300655a03L175.

@singh-surjeet removed that feature on 22nd Match.

I wonder whether bringing previous implementation would make it easier for RPA team. I guess we need to wait for response from them.

Comment (rattur) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: This new version is preferred from the previous implementation, therefore, had conversations and sent emails with the new pdf attached to Clem and RPA team. Will wait for the outcome

**Comment by geek-soft** [comment]
Solution seems valid. I put DO NOT MERGE prefix as we need to confirm with RPA team that change is not going to break automation.

**Comment by geek-soft** [comment]
Solution deployed on RPA side, removing DO NOT MERGE prefix.

**Pull request merged**